src/rtc_memory_esp32.h

```diff
@@ -24,6 +24,8 @@ namespace toit {
 // The RTC memory holds state that is preserved across reboots.
 class RtcMemory {
  public:
+  static const int RTC_USER_DATA_SIZE = 4096;
```

Comment:
In the comment for the Toit functions it says 2K.

Reply:
You asked about the amount of RTC memory we have available. It seems like we have a total of 8K available. We use a bit for boot count and related things, so we could in principle bump this further.

It should be no problem to increase it (maybe we shouldn't have a default value for the length in the toit-level primitive calls if we want to change it).